.github/workflows/R-CMD-as-cran-check.yaml

```diff
@@ -0,0 +1,47 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
```

Comment:
you dont need both of these the difference only exists for tests we don't run on CRAN (usually stan related)

Reply:
ah right now I remember